dbt/models/reporting/schema.yml

```diff
@@ -1,32 +1,157 @@
 sources:
   - name: reporting
-    tags:
-      - load_auto
     tables:
       - name: ratio_stats
-        description: '{{ doc("table_ratio_stats") }}'
```

Comment:
**issue:** The table-level descriptions shouldn't be removed.

Reply:
Mistake as well